### PR TITLE
add .mailmap file to cleanup names in commits

### PR DESCRIPTION
**Pull request by da2ce7**
In this commit I have created a mailmap taken from the command:

```
( git log  --format="%aN <%aE>" --reverse ; git log  --format="%cN <%cE>" --reverse ) | perl -e 'my %dedupe; while (<STDIN>) { print unless $dedupe{$_}++}'
```

Please review the `.mailmap` file included in this pull request.

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Edwin Young <edwin@bathysphere.org>
+Edwin Young <edwin@bathysphere.org> catenary <catenary@users.sourceforge.net>
+Edwin Young <edwin@bathysphere.org> edwin young <catenary@edwins-MacBook-Air.local>
+Edwin Young <edwin@bathysphere.org> Tim Whidbey <catenary@bathysphere.org>
+Cameron Garnham <me@da2ce7.com>
+Richard Mant <dx-mon@users.sourceforge.net> Richard Mant <dx-mon@DX-TOP.(none)>
```
Comment (DX-MON) [inline .mailmap:6]: Could I get you to change this please as it dead-names me? Thanks

Comment (da2ce7) [reply .mailmap:6]: @DX-MON You would like to be called `dx-mon <dx-mon@users.sourceforge.net>` without the pseudonym 'Richard Mant'?

Comment (da2ce7) [reply .mailmap:6]: Ahh, I get it; now your called: 'Rachel Mant', my apologies. I will make a corrective pull request.

Comment (DX-MON) [reply .mailmap:6]: Correct, thank you :)

Comment (da2ce7) [reply .mailmap:6]: But as always, as you like. Maybe you would prefer just `dx-mon`.

Comment (da2ce7) [reply .mailmap:6]: @DX-MON https://github.com/fract4d/gnofract4d/pull/75 :)

**Pull request merged**